src/schema.ts

```diff
@@ -298,12 +279,8 @@ function resolveMessageOptions(ctx: Context, message: DescriptorProto): Code | u
 
   let messageOptions: Code | undefined;
   if (message.options) {
-    messageOptions = encodedOptionsToOptions(
-      ctx,
-      ".google.protobuf.MessageOptions",
-      (message.options as any)["_unknownFields"]
-    );
-    delete (message.options as any)["_unknownFields"];
+    messageOptions = encodedOptionsToOptions(ctx, ".google.protobuf.MessageOptions", message.options._unknownFields);
+    delete message.options._unknownFields;
```

Comment:
It took me awhile to figure out why this wasn't failing CI....

It looks like our CI process right now doesn't actually type-check until the release step, which is unintentional.

That said, I like this change, so :shrug: I'm going to try and push it through and then follow up with a ts-proto-descriptors bump, as you mentioned in the PR description.